src/assets/fonts.tsx

```diff
@@ -8,4 +8,26 @@ injectGlobal`
         font-weight: normal;
         font-style: normal;
     }
-`
\ No newline at end of file
+`
+
+const primaryFontFamily = `'ITC Avant Garde Gothic W04', 'AvantGardeGothicITCW01D 731075', 'AvantGardeGothicITCW01Dm', 'Helvetica', 'sans-serif'`,
+
+export const primary = {
+    fontFamily: primaryFontFamily,
+    style: `
+        font-family: ${primaryFontFamily};
+        -webkit-font-smoothing: antialiased;
```

Comment:
Yeah good point, styled-components is supposed to add the vendor prefixes for us so let me try to remove it. Hopefully it should work